core/variables.tf

```diff
@@ -8,3 +8,17 @@ variable "secret_key_base" {
   description = "The secret key base for the application"
   type        = string
 }
+
+variable "ecs" {
+  description = "ECS input variables"
+
+  type = object({
+    task_desired_count                 = number
+    web_container_cpu                  = number
+    web_container_memory               = number
+    deployment_maximum_percent         = number
+    deployment_minimum_healthy_percent = number
+    max_capacity                       = number
+    max_cpu_threshold                  = number
+  })
+}
```

Comment:
This setting should be defined as local. In case there are multiple environments need to be supported, a more complex setup can be applied, example: 
https://github.com/nimblehq/easyhotel-infra-aws/blob/4963d066a3e38ff06833840b265660ad65ffdf68/core/locals.tf#L120-L125

Reply:
Thanks for your sharing, it's an intelligent way to manage the locals instead of using the hash in Terraform Cloud.

b96b26d ✅